_source/_posts/2023-07-31-workflows-workshop.md

```diff
@@ -0,0 +1,939 @@
+---
+layout: blog_post
+title: "Enterprise Ready Workshop: Automate with with no-code Okta Workflows"
+author: max-katz
+by: advocate
+communities: [no-code,javascript]
+description: "Add automated reporting with no-code Okta Workflows. Enhance customer satisfaction by enabling custom automation."
+tags: [enterprise-ready-workshops, workflows, no-code, low-code]
+tweets:
+- ""
+image: blog/workflows-workshop/social.jpg
+type: awareness
+---
+
+
+{% include toc.md %}
+
+This tutorial is part of the enterprise-ready workshop series. In this workshop, you'll enhance the base Todo application by creating an automated report using Okta's no-code Workflows platform.
+
+## Introduction
+
+You built a SaaS (Software as a Service) Todo application: 
+
+{% img blog/workflows-workshop/Workflows_todoapp.jpg alt:"The Todo application showing five tasks" width:"800" %}{: .center-image }
+
+The application allows entering todo items and marking them completed when done. 
+
+But it's missing an important enterprise capability. 
+
+You need to add a report that summarizes all the Todo items for your organization and automatically emails the report once a week. 
+
+The email report looks like this: 
+
+{% img blog/workflows-workshop/Workflows_email_report.jpg alt:"Todo email report showing each todo item from the Todo application with a boolean indicating completed state" width:"800" %}{: .center-image }
+
+|Posts in the enterprise-ready workshop series|
```

Comment:
The titles changes in this series, so these links need to be updated

Reply:
I removed the Introduction header and updated the titles.